src/components/Dashboard/index.jsx

```diff
@@ -8,12 +8,46 @@ import listenToSocketEvent from '../../realTime';
 import ActivityFeed from './ActivityFeed';
 import Header from '../Header';
 import PartnerStats from './PartnerStatsCard';
+import ProgressBar from './UpSellingPartners/index';
 
 class Dashboard extends React.Component {
   state = {
     reportData: [],
   };
 
+  partnerStats = [
```

Comment:
@johnkegz Is there any reason `partnerStats` is not part of the `state`?

Reply:
@09arnold this has been solved.